packages/testing/src/test/chai-retry-plugin.unit.ts

```diff
@@ -217,6 +217,30 @@ describe('chai-retry-plugin', () => {
             await expect(resultFunction).retry().to.not.change(myObj, 'comas');
             expect(getCallCount()).to.equal(2);
         });
+
+        it('.respondTo()', async () => {
+            class EchoService {
+                echo: (() => void) | undefined;
+            }
+
+            setTimeout(() => {
+                EchoService.prototype.echo = () => {
```

Comment:
do we actually need this class/prototype stuff? Will `const o = {}; ... o.test = () => null;` work?

Reply:
@yurii-ve yeah, could be simplified, fixed